src/app/dashboard/integrations.component.html

```diff
@@ -138,42 +138,17 @@ <h2 class="card-pf-title">
                      placement="bottom"></i>
                 </div>
 
-
                 <!-- Actions -->
-                <div class="col-xs-1 actions">
-                  <div dropdown
-                       class="dropdown pull-right dropdown-kebab-pf">
-                    <button dropdownToggle
-                            class="btn btn-link"
-                            type="button"
-                            id="dropdownKebabRight9"
-                            aria-haspopup="true"
-                            aria-expanded="true">
-                      <span class="fa fa-ellipsis-v"></span>
-                    </button>
-                    <ul *dropdownMenu
-                        class="dropdown-menu dropdown-menu-right"
-                        aria-labelledby="dropdownKebabRight9">
-                      <li>
-                        <a [routerLink]=" ['/integrations/edit', integration.id, 'save-or-add-step'] ">Edit</a>
-                      </li>
-                      <li *ngIf="integration.currentStatus === 'Activated'">
-                        <a (click)="requestDeactivate(integration)">Deactivate</a>
-                      </li>
-                      <li *ngIf="integration.currentStatus === 'Deactivated'">
-                        <a (click)="requestActivate(integration)">Activate</a>
-                      </li>
-                      <!--
-                      <li>
-                        <a>Duplicate</a>
-                      </li>
-                      -->
-                      <li>
-                        <a (click)="requestDelete(integration)">Delete</a>
-                      </li>
-                    </ul>
-                  </div>
+
+                <div class="col-xs-1">
+                  <syndesis-integration-actions [integration]="integration"
```

Comment:
Not the first attribute `[integration]="integration"`, rather the second like `(activate)="requestActivate($event)"`. In the other file you worked on (`src/app/integrations/actions/actions.component.html`), it's exactly how we typically format it, but again, not a big deal at all. :)